models/Type.js

```diff
@@ -7,3 +7,5 @@ const TypeSchema = new Schema({
 		required: true
 	}
 });
+
+module.exports = Type = mongoose.model('types', TypeSchema);
```

Comment:
same comment as above... we can all talk about it together though

Reply:
`type` changed to `category`